gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/process/ProcessLimitedStandardSuite.java

```diff
@@ -18,105 +18,20 @@
-import org.apache.tinkerpop.gremlin.process.traversal.strategy.optimization.EarlyLimitStrategyProcessTest;
-import org.apache.tinkerpop.gremlin.process.traversal.strategy.optimization.IncidentToAdjacentStrategyProcessTest;
-import org.apache.tinkerpop.gremlin.process.traversal.strategy.verification.ReadOnlyStrategyProcessTest;
 import org.junit.runners.model.InitializationError;
 import org.junit.runners.model.RunnerBuilder;
 
 /**
- * The {@code ProcessLimitedComputerSuite} is a set of tests that providers may implement in addition to the standard Gherkin
- * feature tests. The {@link ProcessStandardSuite} contains all the tests in Gherkin and is therefore duplicative of
- * that test set but also includes a number of tests that have not been (or cannot be) converted to Gherkin for various
- * reasons.
- *
  * @author Stephen Mallette (http://stephen.genoprime.com)
+ * @deprecated As of release 3.8.0, replaced by {@link ProcessEmbeddedStandardSuite}.
  */
-public class ProcessLimitedStandardSuite extends AbstractGremlinSuite {
-
-    /**
-     * This list of tests in the suite that will be executed as part of this suite.
-     */
-    private static final Class<?>[] allTests = new Class<?>[]{
-
-            MergeVertexTest.Traversals.class,
-            MergeEdgeTest.Traversals.class,
-
-            MatchTest.CountMatchTraversals.class,
-            MatchTest.GreedyMatchTraversals.class,
-            ProfileTest.Traversals.class,
-            WriteTest.Traversals.class,
-            ExplainTest.Traversals.class,
-            SideEffectTest.Traversals.class,
-            SubgraphTest.Traversals.class,
-            TreeTest.Traversals.class,
-
-            // compliance
-            ComplexTest.Traversals.class,
-            CoreTraversalTest.class,
-            TraversalInterruptionTest.class,
-
-            // creations
-            TranslationStrategyProcessTest.class,
-
-            // decorations
-            ElementIdStrategyProcessTest.class,
-            EventStrategyProcessTest.class,
-            PartitionStrategyProcessTest.class,
-
-            // optimizations
-            IncidentToAdjacentStrategyProcessTest.class,
-            EarlyLimitStrategyProcessTest.class,
-
-            // semantics
-            TernaryBooleanLogicsTest.class,
-    };
-
-    /**
-     * A list of the minimum set of base tests that Gremlin flavors should implement to be compliant with Gremlin.
-     */
-    private static final Class<?>[] testsToEnforce = new Class<?>[]{
-            ProfileTest.class,
-            SideEffectTest.class,
-            SubgraphTest.class,
-            TreeTest.class,
-    };
-
-    /**
-     * This constructor is used by JUnit and will run this suite with its concrete implementations of the
-     * {@code testsToEnforce}.
-     */
+@Deprecated
+public class ProcessLimitedStandardSuite extends ProcessEmbeddedStandardSuite {
```

Comment:
Do these old "limited" suites have any utility left after stripping out `allTests` and `testsToEnforce`? I'm wondering if we are better off completely removing the old suites instead of leaving them stripped bare and deprecated.

Reply:
the utility is that it gives providers a chance to upgrade without breaks. they can make the change at their discretion rather than face a compile error. i dont think it hurts to leave them in 3.x and remove them in 4.x.